src/test/java/pages/HomePage.java

```diff
@@ -1,20 +1,55 @@
 package pages;
 
 import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
 
-public class HomePage extends BasePage {
+import java.time.Duration;
 
-    By userAvatarIcon = By.cssSelector("img.avatar");
+public class HomePage extends BasePage {
 
-    public HomePage( WebDriver givenDriver) {
+    static WebDriverWait wait;
 
+    public HomePage(WebDriver givenDriver) {
         super(givenDriver);
     }
 
-    public WebElement getUserAvatar () {
-        return findElement(userAvatarIcon);
+    By addAPlaylist = By.xpath("//*[@class='fa fa-plus-circle create']");
+    By clickNewPlaylist = By.xpath("//*[@data-testid='playlist-context-menu-create-simple']");
+    By playlistName = By.cssSelector("input[placeholder='↵ to save']");
+    By clickPlaylist = By.cssSelector(".playlist:nth-child(3)");
+    By chooseName = By.cssSelector("input[name='name']");
+
+    public void addPlaylist() {
+
+        driver.findElement(addAPlaylist).click();
+    }
+
+    public void newPlaylist() {
+        driver.findElement(clickNewPlaylist).click();
+    }
+
+    public void namePlaylist() {
+        driver.findElement(playlistName).clear();
+        driver.findElement(playlistName).click();
+        driver.findElement(playlistName).sendKeys("best songs");
+        driver.findElement(playlistName).sendKeys(Keys.ENTER);
+    }
+
+    public void doubleClickToPlaylist() {
+        doubleClick(driver.findElement(clickPlaylist));
+    }
+
+    public static void choosePlaylist() {
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".playlist:nth-child(3)"))).click();
```

Comment:
Please, remove code redundancies

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )

For example

  ` static By clickPlaylist = By.cssSelector(".playlist:nth-child(3)");`
  
      public static void choosePlaylist() {
        wait.until(ExpectedConditions.visibilityOfElementLocated(clickPlaylist)).click();
    }